operator_ui/install.go

```diff
@@ -0,0 +1,165 @@
+package main
+
+import (
+	"archive/tar"
+	"compress/gzip"
+	"context"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"path"
+	"path/filepath"
+	"strings"
+	"time"
+)
+
+func main() {
+	const (
+		owner                  = "smartcontractkit"
+		repo                   = "operator-ui"
+		fullRepo               = owner + "/" + repo
+		tagPath                = "operator_ui/TAG"
+		unpackDir              = "core/web/assets"
+		downloadTimeoutSeconds = 10
+	)
+	rootDir := rootDir()
+
+	tag := mustReadTagFile(path.Join(rootDir, tagPath))
+	strippedTag := stripVersionFromTag(tag)
+	assetName := fmt.Sprintf("%s-%s-%s.tgz", owner, repo, strippedTag)
+	downloadUrl := fmt.Sprintf("https://github.com/%s/releases/download/%s/%s", fullRepo, tag, assetName)
+
+	// Assuming that we're in "root/operator_ui/"
+	unpackPath := filepath.Join(rootDir, unpackDir)
+	err := rmrf(unpackPath)
+	if err != nil {
+		panic(err)
```

Comment:
`log.Fatal*` would exit a little more cleanly
```suggestion
		log.Fatalln(err)
```